controllers/fenceagentsremediation_controller.go

```diff
@@ -224,7 +225,7 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		!meta.IsStatusConditionTrue(far.Status.Conditions, commonConditions.SucceededType) {
 		// Fence agent action succeeded, and now we try to remove workloads (pods and their volume attachments)
 		r.Log.Info("Manual workload deletion", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
-		if err := utils.DeleteResources(ctx, r.Client, req.Name); err != nil {
```

Comment:
do we need to remove also the function implementation? It doesn't seem used elsewhere anymore

Reply:
>do we need to remove also the function implementation?

Are you referring to `utils.DeleteResources`? I have deleted this logic, minimized it, and then moved to [Common](https://github.com/medik8s/common/pull/12) as `DeletePods`.

Reply:
Sorry, I don't know why, but at the beginning I couldn't see the changes in `pkg/utils` :+1: